### PR TITLE
fixed 404 link errors and minor edits

### PR DESCRIPTION
**Pull request by Bradben**
end-to-end.md:
-- fix 404 link errors and minor edits

broombridge.md
-- fix 404 link errors
-- fix generic anchor text instances ("here") and surrounding content as needed
-- Edited the data generation paragraph. The original run-on sentence was hard to interpret - please review to ensure that I didn't alter the intent.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/chemistry/samples/end-to-end.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1173d3432a4b93f68212cb84f65c9c5ef2400aa/articles/libraries/chemistry/samples/end-to-end.md) | 75 | 55 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84ed75d5-7e11-4077-ab16-9cfd94f326d2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/b1173d3432a4b93f68212cb84f65c9c5ef2400aa/articles/libraries/chemistry/schema/broombridge.md) | 68 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b58a992f-20b4-4101-b3ee-38c8fa5ff187_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/chemistry/samples/end-to-end.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/750348456cc45a9052d46910fee958415a410c92/articles/libraries/chemistry/samples/end-to-end.md) | 75 | 55 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89c98a50-b3e2-4266-9753-3262e77e2b93_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/750348456cc45a9052d46910fee958415a410c92/articles/libraries/chemistry/schema/broombridge.md) | 68 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9742c760-5209-415a-9a21-8d4274159917_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/libraries/chemistry/samples/end-to-end.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For more information:
 If you haven't already done so, clone the [Microsoft/Quantum repository](https://github.com/Microsoft/Quantum), which contains samples and utilities for working with the Quantum Development Kit:
 
 ```powershell
-git clone https://github.com/Microsoft/Quantum
+git clone https://github.com/Microsoft/Quantum
```
Comment (cgranade) [inline articles/libraries/chemistry/samples/end-to-end.md]:
```suggestion
git clone https://github.com/Microsoft/Quantum
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/chemistry/samples/end-to-end.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2cdd39ad3a86d2b23d33a9271e693831ca6a9f10/articles/libraries/chemistry/samples/end-to-end.md) | 75 | 55 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7aaf26c3-e567-4418-95c0-6337b3d40564_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2cdd39ad3a86d2b23d33a9271e693831ca6a9f10/articles/libraries/chemistry/schema/broombridge.md) | 68 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5aef69c-a70b-4c4f-b4e1-234d932513ab_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/libraries/chemistry/samples/end-to-end.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2cdd39ad3a86d2b23d33a9271e693831ca6a9f10/articles/libraries/chemistry/samples/end-to-end.md) | 75 | 55 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/489e97c9-dad8-4585-bb08-6cb9ebe8e3ac_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2cdd39ad3a86d2b23d33a9271e693831ca6a9f10/articles/libraries/chemistry/schema/broombridge.md) | 68 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/08856796-2e47-4bc0-8ee0-c8cd20fdd5b8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**